master_data/business_logic/logic_vendor_category.py

```diff
@@ -1,9 +1,8 @@
 from django.db.models import QuerySet
 
-from master_data.models import VendorCategory
+from master_data.models.vendor_category import VendorCategory
 
 
 def get_vendor_categories() -> QuerySet[VendorCategory]:
 
-
     return VendorCategory.objects.filter(deleted_flag=False)
```

Comment:
Add one more filter based on whether a vendor is visible for public or not

Reply:
This filter is in `VendorCategory` or `Vendor`?

Reply:
My mistake, you are right. This filter is for the Vendor, not vendor category.